entity-framework/core/providers/sqlite/index.md

```diff
@@ -29,7 +29,7 @@ Install-Package Microsoft.EntityFrameworkCore.Sqlite
 
 ## Supported Database Engines
 
-* SQLite (3.7 onwards)
+* SQLite (3.46.1 onwards)
```

Comment:
Just curious - how do we decide/test which SQLite version is supported by our EF providers?

Reply:
I don't think we "decide". Before .NET 9 release we took what was latest and referenced the underlying packages. It will probably work with older versions (minus the fixed bugs, etc. like i.e. #34590), but we don't explicitly test this.